ruby/README.md

```diff
@@ -86,6 +85,13 @@ This gem includes the upb parsing and serialization library as a single-file
 amalgamation. It is up-to-date with upb git commit
 `535bc2fe2f2b467f59347ffc9449e11e47791257`.
 
+Alternatively, you can use Bazel to build and to run tests.
+
+From the project root (rather than the `ruby` directory):
+
+    $ bazel build //:protoc
```

Comment:
This isn't necessary, Bazel will build protoc below because it's a dependency

Reply:
Removed